IFC4x3/Properties/s/Symbol_1e6VoJKif46hPnVSUT$o5u/Documentation.md

```diff
@@ -0,0 +1 @@
+Content which is shown on the sign, e.g. text, number, arrow or icon. The string can also be a pointer to a symbol catalog.
```

Comment:
I think I have seen a property that should hold the sign content already. Are many such properties required?